docs/topics/vs-code-extension-reviewing-issues.adoc

```diff
@@ -18,4 +18,4 @@ You can use the {ProductShortName} extension icons to prioritize issues based on
 
 ** image:vs_mandatory.png[Mandatory] : The issue must be fixed for a successful migration.
 ** image:vs_potential.png[Warning] : The issue might need to be addressed during migration.
-** image:vs_optional.png[Optional or Quick Fix] : The issue is optional to fix for migration. This icon is also used to indicate any Quick Fixes available for an issue.
+// ** image:vs_optional.png[Optional or Quick Fix] : The issue is optional to fix for migration. This icon is also used to indicate any Quick Fixes available for an issue.
```

Comment:
This is still present in the preview.

Reply:
![image](https://github.com/windup/windup-documentation/assets/106804821/5a8b7ea4-245b-43a1-b24a-bfc103f5d139)